qutip/core/coefficient.py

```diff
@@ -341,9 +362,9 @@ def try_import(file_name, parsed_in):
             return None, file_name_
 
     if mod.parsed_code != parsed_in:
-        # At least 10 coeff with the same hash!?
-        # Giveup
-        raise ValueError("string hash collision, change the string or ")
+        # At least 10 coeff with the same hash!? Giveup...
+        raise ValueError("string hash collision, change the string "
+                         "or clean files in qutip.settings.install['tmproot']")
```

Comment:
Did we change our minds about allowing hash collisions at all at some point?

Reply:
This was changed in the core branch and not merged here. No new changes to coefficient.